x-pack/plugins/index_management/public/sections/index_list/components/index_table/index_table.js

```diff
@@ -406,6 +416,21 @@ export class IndexTableUi extends Component {
                   onChange={this.onFilterChanged}
                 />
               </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiButton
+                  color="secondary"
+                  onClick={() => {
+                    loadIndices();
```

Comment:
Do we want to call `loadIndices` or `reloadIndices`?

Reply:
nope, loadIndices is correct here.  Reload indices takes a list of index names and only reloads those.  This reload button is intended to refresh the entire list, including picking up new ones.